src/deq_eid/config.py

```diff
@@ -0,0 +1,108 @@
+"""
+config.py: Configuration values. Secrets to be handled with Secrets Manager
+"""
+
+import logging
+import socket
+import urllib
+
+SKID_NAME = "deq-eid"
+
+#: Try to get project id from GCP metadata server for hostname. If it's empty or errors out, revert to local hostname
+try:
+    url = "http://metadata.google.internal/computeMetadata/v1/project/project-id"
+    req = urllib.request.Request(url)
+    req.add_header("Metadata-Flavor", "Google")
+    project_id = urllib.request.urlopen(req).read().decode()
+    if not project_id:
+        raise ValueError
+    HOST_NAME = project_id
+except Exception:
+    HOST_NAME = socket.gethostname()
```

Comment:
**thought**: this could be converted to an environment variable on the function. Although, I'm fairly certain `GOOGLE_CLOUD_PROJECT` is a runtime environment variable set automatically by gcp on gen 1 and 2 functions and run jobs and services.

**suggestion**: we could also document if the HOST_NAME is local and only send emails from the cloud.